docs/developer/app-store/apps/adyen.mdx

```diff
@@ -0,0 +1,176 @@
+
+In addition to processing payments, the Saleor App Payment Adyen provides merchants with tools for managing refunds and chargebacks. Overall, the Saleor App Payment Adyen is a powerful tool for merchants looking to streamline their payment processes and offer their customers a seamless and secure online shopping experience.
+
+:::caution
+
+To successfully configure the Adyen App, you must have an account with [Adyen](https://www.adyen.com).
+
+:::
+
+## Capabilities
+
+The Adyen App implements the following [Saleor sync webhooks related to transactions](../../extending/apps/synchronous-webhooks/transaction-webhooks):
+
+- [`PAYMENT_GATEWAY_INITIALIZE_SESSION`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum#code-style-fontweight-normal-webhookeventtypesyncenumbpayment_gateway_initialize_sessionbcode)
+- [`TRANSACTION_INITIALIZE_SESSION`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum.mdx#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_initialize_sessionbcode)
+- [`TRANSACTION_PROCESS_SESSION`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum.mdx#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_process_sessionbcode)
+- [`TRANSACTION_CHARGE_REQUESTED`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_charge_requestedbcode)
+- [`TRANSACTION_CANCEL_REQUESTED`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_cancelation_requestedbcode)
+- [`TRANSACTION_REFUND_REQUESTED`](../../../api-reference/webhooks/enums/webhook-event-type-sync-enum#code-style-fontweight-normal-webhookeventtypesyncenumbtransaction_refund_requestedbcode)
+
+Furthermore, it's also prepared to handle [async Adyen webhooks](https://docs.adyen.com/development-resources/webhooks).
+
+Adyen App follows the flow described in detail in the [Saleor Payment App documentation](../../payments#saleor-payment-app).
+
+## Configuration
+
+For Adyen to appear as [available payment gateway](../../checkout#listing-available-payment-gateways), you need to install and configure it in the Saleor Dashboard. You need to obtain the API key from Adyen and paste it into the Adyen App configuration form. Then, a wizard will guide you through the process of configuring the Adyen App, generating the HMAC key, and setting up the webhook.
+
+## Usage in Storefront or apps
+
+:::info
+
+Adyen App uses Adyen [Checkout API v70](https://docs.adyen.com/api-explorer/#/CheckoutService/v70/overview) and [Management API v1](https://docs.adyen.com/api-explorer/#/ManagementService/v1/overview).
```

Comment:
IMHO, this should be placed at the beginning of this file. The overview sounds better.

Reply:
Also, does Adyen App works only with Adyen drop-in? I think that it would be worth to add something about this in overview section